src/components/TheWalletConnectModal.vue

```diff
@@ -27,6 +29,8 @@ const wallets = computed<Wallet[]>(() => {
     },
   ]
 })
+
+const connectWalletGuideUrl = CONFIG.uriGuide + '#connect-wallet'
```

Comment:
I think `#connect-wallet` should not be here. It is like a hardcode. Please consider moving this part of the URI to the config too.

Reply:
Done